content/docs/concepts/components_description.md

```diff
@@ -0,0 +1,202 @@
++++
+title = "NSM components"
+weight = 4
+summary = "Network Service Mesh (NSM): The Hybrid/Multi-cloud IP Service Mesh"
++++
+
+## NSM Services
```

Comment:
Perhaps `NSM gRPC Services`? Otherwise I think there is confusion. We use the word `Service` in many contexts